ImageFilterEditor/ImageFilterEditor/Filtering/ApplyFilter/SliderControlsViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SliderControlsViewController.swift
+//  ImageFilterEditor
+//
+//  Created by Shawn Gee on 5/4/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
+import UIKit
+
+protocol SliderControlsDelegate {
+    func sliderAt(index: Int, didChangeValueTo value: Double)
+}
+
+class SliderControlsViewController: UIViewController {
+    
+    // MARK: - Public Properties
+    
```

Comment:
Ah, so it seems like you'll be fetching the filterControls out of the `ImageFilter` object and holding them here?